modules/src/ics02_client/client_def.rs

```diff
@@ -282,6 +284,23 @@ pub enum AnyConsensusState {
 }
 
 impl AnyConsensusState {
+    pub fn latest_timestamp(&self) -> u64 {
+        match self {
+            Self::Tendermint(cs_state) => {
+                let date = <DateTime<Utc> as From<Time>>::from(cs_state.timestamp);
```

Comment:
I think it's usually a good practice to avoid the syntax `<A as B>::method()`. This could make it easier to read:

```suggestion
                let date: DateTime<Utc> = cs_state.timestamp.into();
```